service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/cmo/ReviewCMOController.java

```diff
@@ -117,6 +121,15 @@ public void handleSubmitted(@RequestBody CallbackRequest callbackRequest) {
         if (!cmosReadyForApproval.isEmpty()) {
             if (!JUDGE_REQUESTED_CHANGES.equals(caseData.getReviewCMODecision().getDecision())) {
                 CaseManagementOrder sealed = reviewCMOService.getLatestSealedCMO(caseData);
+                DocumentReference documentToBeSent = sealed.getOrder();
+
+                coreCaseDataService.triggerEvent(
```

Comment:
nice to have: it would be nice to have service that encasulates this with method
sentByPost(DocumentReference)